features/verblijfstitelhistorie.feature

```diff
@@ -1,149 +1,333 @@
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding.code, datumEinde, datumIngang |
+    | datumVan            | 2001-01-01            |
+    | datumTot            | 2002-01-02            |
+    Dan heeft de response geen verblijfstitelhistorie
+
+Rule: Als er meerdere al dan niet historische verblijfstitels elkaar in geldigheidsperiode overlappen dan wordt alleen de verblijfstitel met de m,eest recente datum opneming opgenomen in de response.
+
+  @gba
+  Scenario: Als er meerdere (historische) verblijfstitels zijn met dezelfde ingangsdatum, dan wordt alleen de meest de meest recente opgenomen in de response. Dat is de meest recente datum opneming, of eerste/bovenste in GBA-V antwoord.
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | naam                | waarde                |
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding.code, datumEinde, datumIngang |
+    | datumVan            | 2003-01-01            |
+    | datumTot            | 2005-01-01            |
+    Dan heeft de response een verblijfstitelhistorie met de waarden
+    | aanduiding.code  | datumIngang | datumEinde |
+    | 33               | 20030327    | 20050327   |
+
+  @gba
+  Scenario: Als er een verblijfstitel geldig is over de hele geldigheid van een eerder opgevoerde verblijfstitel, dan wordt alleen de meest de meest recente opgenomen in de response. Dat is de meest recente datum opneming, of eerste/bovenste in GBA-V antwoord.
+    Als de verblijfstitelhistorie wordt geraadpleegd met de volgende parameters
+    | type                | RaadpleegMetPeildatum |
+    | burgerservicenummer | 999991553             |
+    | fields              | aanduiding.code, datumEinde, datumIngang |
+    | datumVan            | 2001-01-01            |
+    | datumTot            | 2002-01-01            |
+    Dan heeft de response een verblijfstitelhistorie met de waarden
+    | aanduiding.code  | datumIngang | datumEinde |
+    | 29               | 20011101    | 20010801   |
+
+Rule: Als de datum einde geldigheid van een verblijfstitel leeg of onbekend is en er is een volgende verblijfstitel, dan wordt voor de eerste verblijfstitel datumEinde gevuld met de ingangsdatum van de volgende verblijfstitel.
+  @gba
+  Scenario: Raadpleeg de verblijfstitelhistorie met vervallen of ingetrokken verblijfstitel
+    Gegeven het systeem kent een persoon met de volgende gegevens
+    | burgerservicenummer |
+    | 999995182           |
+    En de persoon heeft de volgende verblijfstitel-gegevens
+    | categorie | Aanduiding (39.10) | Datum einde (39.20) | Ingangsdatum (39.30) | Datum geldigheid (85.10) |
+    | 10        | 42                 | 20190901            | 20160901             | 20160905                 |
+    | 60        | 26                 | 00000000            | 20091129             | 20091203                 |
```

Comment:
een volledig onbekende einddatum is wel erg exotisch. Een verblijfstitel wordt door de Nederlandse overheid toegekend. Afwezig zijn van een einddatum daarentegen is heel normaal, bij een vergunning voor onbepaalde tijd. Dus ik zou dat in het voorbeeld gebruiken.